packages/http-client-csharp/emitter/src/sdk-context.ts

```diff
@@ -29,6 +29,12 @@ import {
 } from "./type/input-type.js";
 import { OperationResponse } from "./type/operation-response.js";
 
+/**
+ * Provides the default SDK context used throughout the application.
+ * @beta
+ */
+export let defaultSdkContext: SdkContext<CSharpEmitterOptions, SdkHttpOperation> = undefined as any;
```

Comment:
Where do we need to use this? Should we pass it in the callback instead?

Reply:
We need to use this in mgmt emitter as explained in the PR description, we'd better to share  the same created sdkContext, it's quite to create a new one.

And actually you are right, we can pass it in, where is is needed https://github.com/microsoft/typespec/blob/58e791b819072e69fd5f8576ed88a3f384cb2ff9/packages/http-client-csharp/emitter/src/options.ts#L161